src/unit_tests/os_crypto/blowfish/test_blowfish_op.c

```diff
@@ -14,7 +14,6 @@
 
 #include "../headers/shared.h"
 #include "../../os_crypto/blowfish/bf_op.h"
-#include "../../wrappers/common.h"
```

Comment:
Is this modification necessary?

Reply:
Cleaning unnecessary headers improves compilation times.

Reply:
Sure, but I think it is not part of the issue you are working on. If it is needed to clean up the code, then a specific issue for this should be created.

Reply:
done